ui/src/pages/execution/Execution.jsx

```diff
@@ -213,26 +309,37 @@ export default function Execution() {
       </div>
       <Drawer
         className={classes.drawer}
-        variant="persistent"
+        variant={state.drawerVarient}
         anchor="right"
         open={!!selectedTask}
+        transitionDuration={0}
         classes={{
           root: classes.drawer,
           paper: classes.drawerPaper,
         }}
       >
-        <div className={classes.drawerHeader}>
-          <IconButton onClick={() => handleSelectedTask(null)}>
-            <CloseIcon />
-          </IconButton>
-        </div>
-        <Divider />
-        <RightPanel
-          className={classes.drawerContent}
-          selectedTask={selectedTask}
-          dag={dag}
-          onTaskChange={handleSelectedTask}
-        />
+        <div
```

Comment:
nit: indentation seems a little off here. 

Side note: :if we don't have `prettier` for the project we should think about implementing (maybe in a git hook of some sort)

Reply:
fixed in other PR #2548